app/javascript/dashboard/modules/search/components/SearchView.vue

```diff
@@ -21,39 +21,44 @@
         />
       </header>
       <div class="search-results">
-        <woot-loading-state v-if="uiFlags.isFetching" :message="'Searching'" />
-        <div v-else>
-          <div v-if="all.length">
-            <search-result-contacts-list
-              v-if="filterContacts"
-              :contacts="contacts"
-              :query="query"
-            />
-            <search-result-messages-list
-              v-if="filterMessages"
-              :messages="messages"
-              :query="query"
-            />
-            <search-result-conversations-list
-              v-if="filterConversations"
-              :conversations="conversations"
-              :query="query"
-            />
-          </div>
-          <div v-else-if="showEmptySearchResults && !all.length" class="empty">
-            <fluent-icon icon="info" size="16px" class="icon" />
-            <p class="empty-state__text">
-              {{ $t('SEARCH.EMPTY_STATE_FULL', { query }) }}
-            </p>
-          </div>
-          <div v-else class="empty text-center">
-            <p class="text-center margin-bottom-0">
-              <fluent-icon icon="search" size="24px" class="icon" />
-            </p>
-            <p class="empty-state__text">
-              {{ $t('SEARCH.EMPTY_STATE_DEFAULT') }}
-            </p>
-          </div>
+        <div v-if="all.length">
+          <search-result-contacts-list
+            v-if="filterContacts"
+            :is-fetching="uiFlags.contact.isFetching"
+            :contacts="contacts"
+            :query="query"
+            :show-title="selectedTab === 'all'"
```

Comment:
Can we move it to a computed property? We are using it in multiple places.
```suggestion
            :show-title="isSelectedTabAll"
```